app/src/main/java/com/example/bigowlapp/model/Attendance.java

```diff
@@ -32,20 +32,20 @@ public void setAuthenticated(boolean authenticated) {
         this.authenticated = authenticated;
     }
 
-    public boolean isAuthAttemptedUserMobileNumber() {
-        return authAttemptedUserMobileNumber;
+    public boolean attemptedAuthByUserMobileNumber() {
+        return attemptedAuthByUserMobileNumber;
     }
 
-    public void setAuthAttemptedUserMobileNumber(boolean authAttemptedUserMobileNumber) {
-        this.authAttemptedUserMobileNumber = authAttemptedUserMobileNumber;
+    public void setAttemptedAuthByUserMobileNumber(boolean authAttemptedUserMobileNumber) {
+        this.attemptedAuthByUserMobileNumber = authAttemptedUserMobileNumber;
     }
 
-    public boolean isAuthAttemptedPhoneUid() {
-        return authAttemptedPhoneUid;
+    public boolean attemptedAuthByPhoneUid() {
+        return attemptedAuthByAppUid;
```

Comment:
Not the cause for case 2, but can cause some problems similar to case 2 down the line.

Apply the same solution as suggested above.

Reply:
ok, it's done